### PR TITLE
bumping image tag and change in load_defination

### PR DESCRIPTION
**Pull request by ommkar-xl**

### CODE REVIEW

File: Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: v9.7
```
Comment (spaliwal-xl) [inline Chart.yaml:2]: Update the appVersion to 10.0

Comment (ommkar-xl) [reply Chart.yaml]: done

**Review by spaliwal-xl — APPROVED** [review]
#22 LGTM

**Review by chandras-xl — APPROVED** [review]
#22 LGTM

**Pull request merged**